ui/app/templates/vault/cluster/secrets/backend/list.hbs

```diff
@@ -1,10 +1,10 @@
 {{secret-list-header isCertTab=(eq tab "certs") model=backendModel baseKey=baseKey backendCrumb=backendCrumb filter=filter}}
 
 {{#with (options-for-backend backendType tab) as |options|}}
-  {{#if model.meta.total}}
-    <div class="box is-sideless has-background-grey-lighter has-short-padding is-marginless">
-      <div class="level">
-        <div class="level-left">
+  {{#if (or model.meta.total (or (or (eq model.type "aws") (eq model.type "ssh") (eq model.type "pki")) (not (or isCertTab isConfigure))))}}
```

Comment:
Heh, might be good to move this to the controller in the future so it's a single `if foo` statement.